docs/source/1.0/spec/core/model.rst

```diff
@@ -1,223 +1,431 @@
+are written using the :ref:`Smithy interface definition language (IDL) <idl>`
+syntax. Complementary :ref:`JSON AST <json-ast>` examples are provided
+alongside Smithy IDL examples where appropriate. For example:
 
 .. tabs::
 
     .. code-tab:: smithy
 
+        $version: "1.0"
+
+        metadata foo = "bar"
+
         namespace smithy.example
 
-        string MyString
+        use smithy.other.namespace#MyString
+
+        structure MyStructure {
+            @required
+            foo: MyString
+        }
 
     .. code-tab:: json
 
         {
             "smithy": "1.0",
+            "metadata": {
+                "foo": "bar"
+            },
             "shapes": {
-                "smithy.example#MyString": {
-                    "type": "string"
+                "smithy.example#MyStructure": {
+                    "type": "structure",
+                    "members": {
+                        "foo": {
+                            "target": "smithy.other.namespace#MyString",
+                            "traits": {
+                                "smithy.api#required": {}
+                            }
+                        }
+                    }
                 }
             }
         }
 
 
-.. _use-statement:
+.. _smithy-overview:
+
+---------------
+Smithy overview
+---------------
+
+Smithy is a framework that consists of a semantic model, file formats used to
+define a model, and a build process used to validate models and facilitate
+model transformations.
+
+.. figure:: images/smithy-overview.svg
+   :name: figure-1.1
+   :width: 100%
+   :alt: Smithy framework concepts
+
+   **Figure 1.1**: Smithy framework concepts
+
+Semantic model
+    The in-memory model used by tools. The :ref:`semantic model <semantic-model>`
+    may be serialized into one or more model file representations.
+Model File
+    A file on the file system, in a particular representation. The model files
+    that make up a semantic model MAY be split across multiple files to
+    improve readability or modularity, and those files are not required to
+    use the same representation.
+Representation
+    A particular model file format such as the Smithy IDL or JSON AST.
+    Representations are loaded into the semantic model by mapping the
+    representation to concepts in the semantic model.
+
+    * The :ref:`Smithy IDL <idl>` is a human-readable format that aims to
+      streamline authoring and reading models.
+    * The :ref:`JSON AST <json-ast>` aims to provide a more machine-readable
+      format to easily share models across language implementations and better
+      integrate with JSON-based ecosystems.
+Model transformation
+    The process of converting input model files into output artifacts.
+    Generated artifacts can be other Smithy models (an endogenous
+    transformation), other model formats (an exogenous transformation),
+    code, documentation, etc.
+
+    .. figure:: images/smithy-transformations.svg
```

Comment:
Outdent this, it seems odd to have a figure inside the bulleted list

Reply:
Yeah, I'll dedent and give this it's own dedicated section following this one.